views/js/runner/plugins/security/blurPause.js

```diff
@@ -59,14 +60,20 @@ define([
                 var states = testRunner.getTestData().states;
                 if (!bluring && context.state <= states.interacting && !testRunner.getState('finish')) {
                     bluring = true;
-                    testRunner.trigger('blur')
-                              .trigger('pause', {
-                                  message: lostFocusMessage,
-                                  reasons : {
-                                      category : __('examinee'),
-                                      subCategory : __('behaviour')
-                                  }
-                              });
+                    testRunner.trigger('blur');
+                    if (context.securePauseStateRequired) {
+                        testRunner.trigger('pause', {
+                            message: lostFocusPauseMessage,
+                            reasons : {
+                                category : __('examinee'),
+                                subCategory : __('behaviour')
+                            }
+                        });
+                    } else {
+                        testRunner.trigger('traceLog', {
```

Comment:
Please send only the warning and listen for `blur` using the probe system.
Please reset the value of `bluring` to `false` in this situation, otherwise next attempts won't be recorded.